renga_deployer/config.py

```diff
@@ -65,6 +65,14 @@
 DEPLOYER_K8S_USE_INGRESS = False
 """Whether to use an ingress for reaching k8s endpoints."""
 
+DEPLOYER_K8S_INGRESS_CLASS = 'nginx'
```

Comment:
From practical point of view, can we join `DEPLOYER_K8S_USE_INGRESS` and `DEPLOYER_K8S_INGRESS_CLASS` configuration options?

```
DEPLOYER_K8S_INGRESS = None  # or False disables ingress (default)
DEPLOYER_K8S_INGRESS = 'nginx'  # would enable ingress with 'nginx' class
```

Reply:
make sense. I'll change it.